src/Sylius/Component/Product/Model/AttributeValueTranslation.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Product\Model;
+
+use Sylius\Component\Attribute\Model\AttributeValueTranslation as BaseAttributeValueTranslation;
+
+/**
+ * @author Salvatore Pappalardo <salvatore.pappalardo82@gmail.com>
+ */
+class AttributeValueTranslation extends BaseAttributeValueTranslation
```

Comment:
I think this should be necessary to follow standard [sylius_product_{entity}](https://github.com/Sylius/Sylius/pull/4512/files#diff-05f674233e44f27cbc2360a0b58b77edR16)
as you can see for other Entities too (e.g [Archetype.orm.xml](https://github.com/xantrix/Sylius/blob/issue-4341/src/Sylius/Bundle/ProductBundle/Resources/config/doctrine/model/Archetype.orm.xml) [ArchetypeTranslation.php](https://github.com/xantrix/Sylius/blob/issue-4341/src/Sylius/Component/Product/Model/ArchetypeTranslation.php) )